webpack/prod/postcss.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = ({
         }
       }),
       destination: 'public/styleguide/',
-      index: 'index.html',
+      index: 'index.template.html',

```

Comment:
This index file is not the same as the app index, but the index in /styleguide , we should keep the original name.
The changes on the template look good.